content/md/en/docs/tutorials/smart-contracts/prepare-your-first-contract.md

```diff
@@ -59,87 +63,62 @@ To update your development environment:
 
-1. Open the downloaded file and extract the contents to a working directory.
+Modern releases of `gcc`, `clang`, as well as Visual Studio 2019+ should work.
 
-Alternatively, you can build the preconfigured `contracts-node` manually by running `cargo install contracts-node` on your local computer.
-If you can't download the precompiled node, you can compile it locally with a command similar to the following:
+1. Add the `rust-src` compiler compone
 
 ```bash
-cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git --tag <latest-tag> --force --locked
+rustup component add rust-src
 ```
 
-You can find the latest tag to use on the [Tags](https://github.com/paritytech/substrate-contracts-node/tags) page.
-
-## Install additional packages
-
-After downloading or compiling the `contracts-node` package, you need to install two additional packages:
-
-- The WebAssembly **binaryen** package for your operating system to optimize the WebAssembly bytecode for the contract.
-- The `cargo-contract` command line interface you'll use to set up smart contract projects.
-
-### Install the WebAssembly optimizer
-
-To install the **binaryen** package:
+1. Install the latest version of `cargo-contract`
 
-1. Open a terminal shell on your computer.
-
-1. Use the appropriate package manager for your operating system to install the package.
-
-   For example, on Ubuntu or Debian, run the following command:
+```bash
+cargo install --force --locked cargo-contract --version 2.0.0-rc
+```
 
-   ```bash
-   sudo apt install binaryen
-   ```
+1. Verify the installation and explore the commands available by running the following command:
 
-   On macOS, run the following command:
+```bash
+cargo contract --help
+```
 
-   ```bash
-   brew install binaryen
-   ```
+## Install the Substrate Contracts Node
 
-   For other operating systems, you can download the `binaryen` release directly from [WebAssebly releases](https://github.com/WebAssembly/binaryen/releases).
+To simplify this tutorial, you can [download](https://github.com/paritytech/substrate-contracts-node/releases) a
+precompiled Substrate node for Linux or macOS.
 
-### Install the cargo-contract package
+The precompiled binary includes the FRAME pallet for smart contracts by default.
 
-After you've installed the WebAssembly `binaryen` package, you can install the `cargo-contract` package.
-The `cargo-contract` package provides a command-line interface for working with smart contracts using the ink! language.
+To install the contracts node on macOS or Linux:
 
-1. Open a terminal shell on your computer.
+1. Open the [Releases](https://github.com/paritytech/substrate-contracts-node/releases) page.
 
-2. Install `cargo-dylint` to check ink! contracts and warn you about issues that might lead to security vulnerabilities.
+1. Download the appropriate compressed archive for your local computer.
```

Comment:
```suggestion
2. Download the appropriate compressed archive for your local computer.
```

Reply:
I believe all these changes are to take advantage of markdown automatically incrementing numbered lists? e.g. 

1. One
1. Two
1. Three

![image](https://user-images.githubusercontent.com/75586/217479624-7111b5d8-6da5-4b9e-8c09-6f10003749ca.png)

Not sure which markdown engine substrate-developers-hub is using but it should do this.

Reply:
Exactly, we use Gatsby, [which does support this](https://www.gatsbyjs.com/docs/reference/markdown-syntax/#ordered).